sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/options/ShareCreateOptions.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.share.options;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.storage.file.share.models.ShareAccessTier;
+import com.azure.storage.file.share.models.ShareRequestConditions;
+
+import java.util.Map;
+
+/**
+ * Extended options that may be passed when creating a share.
+ */
+@Fluent
+public class ShareCreateOptions {
```

Comment:
how about CreateShareOptions?

Reply:
our naming convention has so far been [Resource][Api]Options
Looks like there is a guideline 
https://azure.github.io/azure-sdk/general_design.html#client-interface

This applies to the other comment as well

Reply:
okay, however being that strict is gonna lead to some interesting names...